.github/ISSUE_TEMPLATE/03-documentation.yml

```diff
@@ -18,6 +18,14 @@ body:
   - type: textarea
     attributes:
       label: Documentation request
+  - type: textarea
+    attributes:
+      label: Affected Pages
+      description: |
+          Link to the pages relevant to your documentation change request.
```

Comment:
Since some requests might be for new docs, or be unrelated to existing pages generally (e.g., search), I'd like to keep this optional. Should we add a simple `, if any.` to the end of the description to signal that as well?